backend/core/configuration.py

```diff
@@ -318,6 +319,7 @@ def check(self, data: Text) -> Dict:
                 if rule["neighbors"] == ["*"]:
                     rule["neighbors"] = [-1]
                 rule["mitigation"] = flatten(rule.get("mitigation", "manual"))
+                rule["no-export"] = rule.get("no-export", False)
```

Comment:
the changes proposed in the configuartion are fine (non-intrusive, minimal changes), keep as they are